### PR TITLE
Update chart.md

### PR DESCRIPTION
**Pull request by ajithpv**
Minor fixes on chart.md

Signed-off-by: Ajith P Venugopal <ajithpv@outlook.com>

### CODE REVIEW

File: en/object-types/chart.md [hunk 1]
```diff
@@ -63,17 +63,17 @@ Ticks and texts to ticks can be added.
 `list_of_values` is a string with `'\n'` terminated text (expect the last) with text for the ticks. E.g. `const char * list_of_values = "first\nseco\nthird"`. `list_of_values` can be `NULL`.
 If `list_of_values` is set then `num_tick_marks` tells the number of ticks between two labels.  If `list_of_values` is `NULL` then it specifies the total number of ticks.
 
-Where text are added *major tick lines* are drawn, ot the other places *minor tick lines*. `lv_chart_set_x_tick_length(chart, major_tick_len, minor_tick_len)` sets the length of tick lines on the x axis.
+Where text are added *major tick lines* are drawn, not the other places *minor tick lines*. `lv_chart_set_x_tick_length(chart, major_tick_len, minor_tick_len)` sets the length of tick lines on the x-axis.
```
Comment (ajithpv) [inline en/object-types/chart.md]: @embeddedt , Could you please correct this sentence. I didn't get the context here.

Comment (embeddedt) [reply en/object-types/chart.md]: I agree; that sentence was a bit strange. I rewrote it based on my experience with the chart, so hopefully that's more accurate.

**Comment by embeddedt** [comment]
Thanks!

**Comment by kisvegabor** [comment]
I think my intention with that sentence was:
*Where text are added *major tick lines* are drawn, ~~ot~~ on the other places *minor tick lines*.*
But still odd.

Anyway, thank you for the fixes, it's much clearer now.

**Pull request merged**